### PR TITLE
Specific lane library root

### PR DESCRIPTION
**Pull request by leonardr**
This branch addresses https://jira.nypl.org/browse/SIMPLY-1020. It's now possible to set up the library root so it always redirects to a specific lane.

### CODE REVIEW

File: api/custom_index.py [hunk 1]
```diff
@@ -64,7 +64,7 @@ def for_library(cls, library):
         protocol = integration.protocol
         if not protocol in cls.BY_PROTOCOL:
             raise CannotLoadConfiguration(
-                "Unregistered custom index protocol: %s" % name
+                "Unregistered custom index protocol: %s" % protocol
```
Comment (leonardr) [inline api/custom_index.py]: This is an unrelated bug I found when getting this to work for real.

**Comment by aslagle** [comment]
I don't think this actually solves the problem with Open Bookshelf, since it would also change the root lane in the admin interface. The underlying problem is that we need some lanes that are visible to admins but not patrons.

**Comment by aslagle** [comment]
Maybe it would work if the admin interface was set up to always go to /groups instead of the custom index, but that seems like a hack.

**Comment by leonardr** [comment]
That doesn't sound like a hack... I would say the admin interface should always show the top of the lane hierarchy, even if that's not the server root.

The SimplyE Collection's server root is an OPDS navigation feed, and for that one I would expect the admin interface to show the actual lanes rather than ask me if I'm over 13.

**Comment by aslagle** [comment]
The way the lanes are set up in Open Bookshelf is a hack - the top level lane should actually be the Open Bookshelf lane, I just need somewhere to put the internal admin lanes and there isn't anywhere.

Also, eventually, I want the server root for Open Bookshelf to be a web page rather than an OPDS feed.

**Comment by aslagle** [comment]
That would work better for Open Bookshelf.

**Comment by leonardr** [comment]
Abandoning after a Slack conversation; https://github.com/NYPL-Simplified/circulation/pull/996 is my second attempt at solving this problem.
